js/functions.js

```diff
@@ -2211,3 +2218,134 @@ function getEventDescription(isAerobatics, locationName, minutes) {
+        {'hebName':'אדיר','engName':'adir','audioSrc':""},
+        {'hebName':'עטלף','engName':'atalef','audioSrc':""},
+        {'hebName':'בז','engName':'baz','audioSrc':""},
+        {'hebName':'בוינג747','engName':'boeing747','audioSrc':""},
+        {'hebName':'','engName':'c17','audioSrc':""},
+        {'hebName':'','engName':'airtractor','audioSrc':""},
+        {'hebName':'','engName':'c295','audioSrc':""},
+        {'hebName':'עפרוני','engName':'efroni','audioSrc':""},
+        {'hebName':'עיטם','engName':'eitam','audioSrc':"" },
+        {'hebName':'F-16','engName':'f16','audioSrc':""},
+        {'hebName':'קרנף','engName':'karnef','audioSrc':""},
+        {'hebName':'לביא','engName':'lavi','audioSrc':""},
+        {'hebName':'נחשון','engName':'nahshon','audioSrc':""},
+        {'hebName':'פתן','engName':'peten','audioSrc':""},
+        {'hebName':'רעם','engName':'raam','audioSrc':""},
+        {'hebName':'ראם','engName':'reem','audioSrc':""},
+        {'hebName':'סייפן','engName':'saifan','audioSrc':""},
+        {'hebName':'שרף','engName':'saraf','audioSrc':""},
+        {'hebName':'שחף','engName':'shahaf','audioSrc':""},
+        {'hebName':'שמשון','engName':'shimshon','audioSrc':""},
+        {'hebName':'שמשון בריטי','engName':'shimshon-britian','audioSrc':""},
+        {'hebName':'שמשון אוסטרי','engName':'shimshon-ostrian','audioSrc':""},
+        {'hebName':'סופה','engName':'sufa','audioSrc':""},
+        {'hebName':'צופית','engName':'tzufit','audioSrc':""},
+        {'hebName':'ינשוף','engName':'yanshuf','audioSrc':""},
+        {'hebName':'יסעור','engName':'yasur','audioSrc':""}
+    ];
+    var name ="";
+    //check that its not the same popup that has been closed
+    //var timeCount = 0;
+    function notifyUserIfNear(currentLocation, aircraft) {
+        if (userLoc) {
+            currentLocation = {lon: currentLocation.lng, lat: currentLocation.lat};
+            if(haversineDistance(userLoc,currentLocation) < 2000)
+            {
+                notifiedNearUser = true;
+                //check that its not the same popup that has been closed
+                /*if(name === "")
+                {
+                    console.log("in timer");
+                    var x = setInterval( function () { timeCount++;}, 1000);
+                } */
+                //check that its not the same popup that has been closed - if its not the same aircraft or its has been more then 30 sec
+                //if(timeCount >30 || name !== aircraft.name) {
+                    if(document.getElementById("myModal").style.display === "none" && document.getElementById("gottoVoiceMessagePopup").style.display === "none")
+                    {
+                        //timeCount = 0;
+                        name = aircraft.name;
+                        document.getElementById("gottoVoiceMessagePopup").style.display = "block";
+                        document.getElementById("aircraftName").innerHTML = " מטוס קרב " + name;
```

Comment:
it is not always "מטוס קרב", get its type from aircraft object.

Reply:
@mayPeleg Please insert the plane type via string templates too.